.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- [ ] You're expected to ignore `node_modules` folder and `test.md` file. `test.md` file is not yet created and ignored. Kindly do that.

### Project Requirement
![proj-rqt](https://github.com/axcell-0/hello-microverse/assets/26597432/3521d51c-8448-4529-8735-c54f8b42981e)